packages/shared/src/components/layout/MainLayoutHeader.tsx

```diff
@@ -109,7 +108,7 @@ function MainLayoutHeader({
                   }
                 />
                 {hasNotification && (
-                  <Bubble className="right-0 top-0 -translate-y-1/2 translate-x-1/2 cursor-pointer px-1 shadow-bubble-cabbage">
+                  <Bubble className="-right-1.5 -top-1.5 cursor-pointer px-1">
```

Comment:
Did you mean to remove the `shadow-bubble-cabbage`?

Reply:
Yes, since there's no shadow in figma.